src/explore-education-statistics-admin/test/createMemoryHistoryWithMockedPush.ts

```diff
@@ -0,0 +1,9 @@
+import createMemoryHistory from 'history/createMemoryHistory';
```

Comment:
Looks like MethodologySummary is still the only place that uses this so no need for a separate file for this util necessarily, but it would be really nice if we could have it reusable!

As I said, I tried adding to the `explore-education-statistics-admin/test` folder and including an "@admin-test" alias in `tsconfig.json` but couldn't quite get it to work.

Reply:
Nice one!!